flex-panel-gallery/script.js

```diff
@@ -0,0 +1,23 @@
+const panels = document.querySelectorAll('.panel');
+
+
+const clickedPanel = (e) => {
+    const elem = e.target.closest('.panel')
+    elem.classList.toggle("open");
+}
+const clickedPanelWords = (e) => {
+    const elem = e.target.closest('.panel')
+    for(let child of elem.children){
```

Comment:
```suggestion
if(e.propertyName.includes('flex')){
    for(let child of elem.children){
    }
}
```
Better to move if out of loop. Small performance improvement. Then it will be checked only once.